pkg/tuf/verify.go

```diff
@@ -11,13 +11,33 @@ import (
 	"github.com/theupdateframework/notary/client"
 
 	"github.com/cnabio/signy/pkg/cnab"
+	"github.com/cnabio/signy/pkg/docker"
 )
 
+// GetTargetAndSHA returns the target with roles and the SHA256 of the target file
```

Comment:
nit for the future: when moving bits of code around, it's pretty difficult to understand whether this was just moved, or something else also changed.

Reply:
Yeah, sorry about that, I felt guilty about it, too...